apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+
+<HopperProvider colorScheme="light">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout>
+
+## Detailed Installation
+
+### Components
+
+Hopper Components are React Comtponents, uf you are using another framework, or React Native, you should use the [components tokens](/tokens/getting-started/introduction) directly when developping your own components.
+
+#### Installation
+
+Run one of the following commands in your terminal to install the Hopper Components package.
+
+<PackageInstallation library="components" mod="prod" />
+
+#### Usage
+
+A Provider is required to use Hopper Components. Wrap your application with the HopperProvider component to use our components.
+
+```jsx
+import { Button } from "@hopper-ui/components";
+
+<HopperProvider theme="dark">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+### Tokens
+
+Hopper Tokens are a collection of design tokens that should be used to style your application. [Semantic tokens](#) are available for everyone, where [components tokens](#) are designed with non React users in mind.
+
+#### Installation
+
+<PackageInstallation library="tokens" mod="prod" />
+
+#### Usage
+
+##### CSS
+
+Import the tokens in your project by doing the following:
+
+```css
+@import "@hopper-ui/tokens/tokens.css";
+```
+
+If your application supports a dark mode, import dark mode tokens as well:
+
+```css
+@import "@hopper-ui/tokens/dark/tokens.css";
+```
+
+You can now use Hopper tokens in your project by using the following syntax:
+
+```css
+.my-class {
+    background-color: var(--hop-primary-surface-weak);
+}
+```
+
+##### Fonts
+
+In order to use Hopper fonts in your project, import the following:
+
+```css
+@import "@hopper-ui/tokens/fonts.css";
+```
+
+All the necessary font-face declarations are now imported in your project.
+
+### Icons
+
+#### Installation
+
+<PackageInstallation library="icons" mod="prod" />
+
+#### Usage
+
+You can now use [icons](/icons/react-icons/icon-library) and [rich icons](/icons/react-icons/rich-icon-library) in your project by using the following syntax:
+
+```tsx
+import { AddIcon } from "@hopper-ui/icons";
+import { ConversationRichIcon } from "@hopper-ui/icons";
+
+<AddIcon />
+<ConversationRichIcon />
+```
+
+#### Non-standard installations
```

Comment:
Non-standard is a bit of a strange name, I don't have much inspiration for a title. Maybe **Javascript installations** or **Framework agnostic installations**.